public/index.html

```diff
@@ -0,0 +1,11 @@
+<!Doctype html>
+<html>
+  <head>
+    <title>Testing socket connection</title>
+  </head>
+  <body>
+  <h1>Testing Socket Connection</h1>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/socket.io/2.2.0/socket.io.dev.js'></script>
+  <script src='./index.js'></script>
+  </body>
+</html>
```

Comment:
The title tag of this HTML document is saying "Testing socket connection". Will you be using this HTML document in production?. If we will not be using it, then I don't think this HTML document should be included in this pull request.